environment

```diff
@@ -86,8 +88,10 @@ DSS_GS_CHECKOUT_BUCKET_TEST_USER=org-hca-dss-checkout-test-user  # for tests w/n
 DSS_GS_CHECKOUT_BUCKET_INTEGRATION=org-humancellatlas-dss-checkout-integration
 DSS_GS_CHECKOUT_BUCKET_STAGING=org-humancellatlas-dss-checkout-staging
 DSS_GS_CHECKOUT_BUCKET_PROD=org-humancellatlas-dss-checkout-prod
-DSS_AWS_FLASHFLOOD_PREFIX="aws"
-DSS_GCP_FLASHFLOOD_PREFIX="gcp"
+DSS_AWS_FLASHFLOOD_PREFIX_READ="aws"
+DSS_AWS_FLASHFLOOD_PREFIX_WRITE="aws"  # This can be a comma separated list
+DSS_GCP_FLASHFLOOD_PREFIX_READ="gcp"
+DSS_GCP_FLASHFLOOD_PREFIX_WRITE="gcp"  # This can be a comma separated list
```

Comment:
https://github.com/HumanCellAtlas/data-store/tree/master/docs/environment should probably be updated.

Reply:
updated